placetobe-campaign-page.html

```diff
@@ -541,6 +551,15 @@
         window.open(url,"Share on LinkedIn");
       },
 
+      _computeAvailablesIncentives: function (groupedIncentives, contractTypes) {
+        var groupedIncentives = groupedIncentives.base;
+        var contractTypes = contractTypes.base;
+
+        // first item in contractTypes array is currently the only supported type for PlaceToBe
+        var incentives = groupedIncentives[contractTypes[0].id] || [];
```

Comment:
kan deze niet potentieel een null pointer exception geven? Als in: `contractTypes` kan leeg zijn, en dan is het `id` of `undefined`